### DISCUSSION TITLE
Refresh Rate When Going Out of Low Battery State

### DISCUSSION THREAD

**Discussion by XCJ775** [tribalfs/GalaxyMaxHzPub#248] [General]
When the refresh rate mode is set to "Adaptive" and the phone hits the low battery threshold, it remains at 60 Hz after being charged well above the low battery range, forcing me to go into GalaxyMaxHz and switch from Adaptive to Standard and then back to Adaptive again for it to go back to 120 Hz.

**Comment by tribalfs** [comment]
The app uses the system broadcast/notification signaling that the device is no longer considered in 'low battery' state.   Typically, this occurs when the device has acquired an at least additional 10% charge .
